models/dex/dex_schema.yml

```diff
@@ -5,36 +5,39 @@ models:
     meta:
       blockchain: ethereum
       sector: dex
-      contributors: jeff-dude
+      contributors: jeff-dude, bh2smith
     config:
-      tags: ['ethereum','uniswap','trades', 'dex','cross-chain']
+      tags: ['ethereum', 'trades', 'dexes', 'aggregators', 'cross-chain']
     description: >
-        DEX trades on all chains across all contracts and versions
+      DEX trades on all chains across all contracts and versions including both AMM and aggregator trades
     columns:
+      - &category
+        name: category
+        description: "Whether the trade was settled directly via a 'DEX' (aka AMM) or an 'Aggregator'"
       - &blockchain
         name: blockchain
-        description: "Blockchain which the DEX is deployed"
-      - &project 
+        description: "Blockchain which the project is deployed"
+      - &project
         name: project
-        description: "Project name of the DEX"  
+        description: "Project name"
       - &version
         name: version
-        description: "Version of the contract built and deployed by the DEX project"  
+        description: "Version of the contract built and deployed by the project"
       - &block_date
         name: block_date
-        description: "UTC event block date of each DEX trade"
+        description: "UTC event block date of each trade"
       - &block_time
         name: block_time
-        description: "UTC event block time of each DEX trade"
+        description: "UTC event block time of each trade"
       - &token_bought_symbol
         name: token_bought_symbol
-        description: "Token symbol for token bought in the transaction"
+        description: "Token symbol for token bought in the trade"
```

Comment:
I noticed that the term transaction was being used, but this doesn't generalize to transactions with multiple trades at once, so I changed all occurrences of "transaction" to "trade" which I believe to be the most appropriate term for this (since its a trades table).